sensor_response_curves/__init__.py

```diff
@@ -250,5 +250,9 @@ def get_response_curves(
         bandkeys = ['pan']
     else:
         bandkeys = _get_default_bands(sensor)
-    data_stacked = np.vstack(data[name] for name in bandkeys if name in data.dtype.names)
```

Comment:
I looove list comprehensions, so pls just do

```python
data_stacked = np.vstack([data[name] for name in bandkeys if name in data.dtype.names])
```

The issue before was that it was not a list comprehension (in `[]`) but a generator expression. Those are very neat, too - unless you need a sequence type rather than a generator.